### DISCUSSION TITLE
Twitch Channel Points Miner v2 crashes during login process

### DISCUSSION THREAD

**Discussion by gitgryllus** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#286] [Q&A] [answered]
Hello, I have tried this on two different Win 10 computers and get the same result, even with the old v1 miner. Problem occurs since yesterday. When I open the miner it asks me for login and 2FA. After entering 2FA it says that it can't login: 
"Console login unavailable (CAPTCHA solving required).
What browser do you use? Chrome (1), Firefox (2), Other (3):" 

Then I select Chrome (also tried Firefox) 

Then I get this message:

"Please login inside your browser of choice (NOT incognito mode) and press Enter..."

Then I press enter and the window closes. I managed to take a screenshot of what's happening in this one second, so you can see the error here, that appears before it closes.

![error](https://user-images.githubusercontent.com/91606127/135262781-29306e2b-3e86-49ff-bf7c-2aa468cd430f.PNG)

**Comment by Rakambda** [comment, marked as answer]
Twitch seem to have changed things, this may be related to #281
